content/plugin/sdkv2/debugging.mdx

```diff
@@ -89,25 +89,54 @@ To do so, build the provider binary with the necessary
 
 ### Starting A Provider In Debug Mode
 
-Once a provider has a debug mode added to its `main` function, it can be
-activated. Run your debugger, and pass it the provider binary as the command to
-run, specifying whatever flags, environment variables, or other input is
-necessary to start your provider in debug mode:
+Once a provider has a debug mode added to its `main` function, it can be activated. Run your debugger, which is typically built into an editor or the Delve CLI, by passing it the provider binary as the command to execute. Specify any flags, environment variables, or other input that is necessary to start your provider.
```

Comment:
Just trying to make this be more in active voice - please tweak as necessary if I've misunderstood the meaning! 

```suggestion
After you add a debug mode to your provider's `main` function, you can activate the debugger, which is typically built into an editor or the Delve CLI. To run your debugger, pass it the provider binary as the command to execute. Specify flags, environment variables, and any other input your provider needs to run.
```